### PR TITLE
Add org.gnome.Games.Libretro.PicoDrive

### PR DESCRIPTION
**Pull request by Kekun**
This depends on PR #62.

### CODE REVIEW

**Comment by Kekun** [comment]
I just updated the commit to install the appdata as done here: https://github.com/flathub/org.gtk.Gtk3theme.Adapta/blob/master/org.gtk.Gtk3theme.Adapta.json#L31

File: org.gnome.Games.LibretroPlugin.PicoDrive.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        "cxxflags": "-O2 -g"
+    },
+    "cleanup": ["/include", "/lib/pkgconfig",
+                "/share/pkgconfig", "/share/aclocal",
+                "/man", "/share/man", "/share/gtk-doc",
+                "*.la", "*.a",
+                "/lib/girepository-1.0",
+                "/share/dbus-1", "/share/doc", "/share/gir-1.0"
+    ],
+    "modules": [
```
Comment (Barthalion) [inline org.gnome.Games.LibretroPlugin.PicoDrive.json]: As usually, the `simple` buildsystem would be better here.

**Comment by Barthalion** [comment]
Marking as "blocked" due te dependency on (not merged) Gnome Games.

**Comment by nedrichards** [comment]
GNOME games is now merged

File: org.gnome.Games.LibretroPlugin.PicoDrive.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{
+    "id": "org.gnome.Games.LibretroPlugin.PicoDrive",
+    "runtime": "org.gnome.Games",
+    "runtime-version": "3.24",
```
Comment (nedrichards) [inline org.gnome.Games.LibretroPlugin.PicoDrive.json]: The merged version of Games has updated to the 3.26 version, I assume a mismatch could cause issues?

**Comment by nedrichards** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/938

**Comment by nedrichards** [comment]
Yeah, fails on GNOME Games 3.24 not existing as it's now 3.26.

**Pull request closed without merging**